File: lib/core/config/legal_config.dart
/// Public legal document URLs (Firebase Hosting).
class LegalConfig {
  LegalConfig._();

  static const String _host = 'https://hello-tiktok-57dc5.web.app';

  static String privacyPolicyUrl({String languageCode = 'en'}) =>
      '$_host/legal/privacy.html?lang=${languageCode == 'ar' ? 'ar' : 'en'}';

  static String termsOfServiceUrl({String languageCode = 'en'}) =>
      '$_host/legal/terms.html?lang=${languageCode == 'ar' ? 'ar' : 'en'}';
}

File: hilla_ride_ios/HelloTukTuk/Core/Config/LegalConfig.swift
import Foundation

enum LegalConfig {
    private static let host = "https://hello-tiktok-57dc5.web.app"

    static func privacyPolicyURL(languageCode: String = "en") -> URL {
        let lang = languageCode == "ar" ? "ar" : "en"
        return URL(string: "\(host)/legal/privacy.html?lang=\(lang)")!
    }

    static func termsOfServiceURL(languageCode: String = "en") -> URL {
        let lang = languageCode == "ar" ? "ar" : "en"
        return URL(string: "\(host)/legal/terms.html?lang=\(lang)")!
    }
}
